content/blog/nats-security-update.md

```diff
@@ -31,7 +31,7 @@ maximize coverage, the testers performed a source code audit, as well as
 engaged in classic penetration testing against a NATS-provided cloud
 instance.”_
 
-Emphasis was on the [NATS server](https://github.com/nats-io/gnatsd) and our
+Emphasis was on the [NATS server](https://github.com/nats-io/nats-server) and our
 canonical client, the [Go client](https://github.com/nats-io/go-nats),
```

Comment:
Should update the clients too here. go-nats -> nats.go, node-nats -> nats.js etc.